README.md

```diff
@@ -50,9 +51,10 @@ Docker image source files:
 
 - [cptactionhank/atlassian-jira](https://hub.docker.com/r/cptactionhank/atlassian-jira/)
 - [cptactionhank/atlassian-confluence](https://hub.docker.com/r/cptactionhank/atlassian-confluence/)
-- [atlassian/bitbucket-server](https://hub.docker.com/r/atlassian/bitbucket-server/)
+- [cptactionhank/atlassian-bitbucket](https://hub.docker.com/r/cptactionhank/atlassian-bitbucket/)
```

Comment:
Why we need to prefer  to use docker image of `cptactionhank/atlassian-bitbucket` instead of official `atlassian/bitbucket-server`?